src/main/java/com/pipl/api/search/SearchConfiguration.java

```diff
@@ -251,44 +254,44 @@ public String getMatchRequirements() {
 	 * @param matchRequirements a match requirements criteria.
 	 * This criteria defines what fields must be present in an
 	 * API response in order for it to be returned as a match.
-	 * For example: "email" or "email or phone", or 
+	 * For example: "email" or "email or phone", or
 	 * "email & (phone | name)".
 	 */
 	public void setMatchRequirements(String matchRequirements) {
 		this.matchRequirements = matchRequirements;
 	}
-	
+
 	public String getSourceCategoryRequirements() {
 		return this.sourceCategoryRequirements;
 	}
-	
+
 	/**
 	 * @param sourceCategoryRequirements a source category requirements criteria.
 	 * This criteria defines what source categories must be present in an
 	 * API response in order for it to be returned as a match.
-	 * For example: "email" or "email or phone", or 
+	 * For example: "email" or "email or phone", or
 	 * "media & (personal_profiles | professional_and_business)".
 	 */
 	public void setSourceCategoryRequirements(String sourceCategoryRequirements) {
 		this.sourceCategoryRequirements = sourceCategoryRequirements;
 	}
-	
+
 	public boolean getInferPersons() {
 		if (this.inferPersons) {
 			return false;
 		}
 		return this.inferPersons;
 	}
-	
+
 	/**
-	 * @param InferPersons 
+	 * @param InferPersons
 	 * If set, the API may return person responses made up
 	 * solely from data inferred by statistical analysis.
 	 */
 	public void setInferPersons(boolean inferPersons) {
 		this.inferPersons = inferPersons;
 	}
-	
+
 	@Override
 	public String toString() {
 		StringBuilder sb = new StringBuilder();
```

Comment:
What happened to the indentation here...

Reply:
auto indention by IntelliJ..